### PR TITLE
capstone-project

### PR DESCRIPTION
**Pull request by gilberthappi**
Understand how to use medium-fidelity wireframes to create a UI.
Follow written requirements (e.g. user stories).
Use semantic HTML tags.
Apply best practices in HTML code.
Use CSS selectors correctly.
Use CSS box model.
Use Flexbox to place elements in the page.
Use images and backgrounds to enhance the look of a website.
Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
Use GitHub Pages to deploy web pages.
Apply JavaScript best practices and language style guides in code.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.
Use objects to store and access data.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/06d805300f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Capstone</title>
+</head>
+<body>
+    <div class="header-section1-container">
+        <!-- header -->
+        <header class="header">
```
Comment (BregornOriginal) [inline index.html:14]: - [ ] In your mobile version, kindly add to your home page, the hamburger button as you did on the about page

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    </div>
+    
+    <!-- Main program section -->
+    <section class="section-2" id="program">
+        <div class="h2-red-line">
+            <h2 class="h2">Main Program</h2>
+            <hr class="red-line">
+        </div>
+
+        <div class="container">
```
Comment (BregornOriginal) [inline index.html:80]: - [ ] Kindly modify the height on the containers to match with the behance design

Your design

image

Behance design

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @gilberthappi  

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

***Your project looks great! Excellent work!***

giphy (5)

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (gilberthappi) [reply index.html:14]: Okey 
thanks

File: index.html (hunk 2 above)
Comment (gilberthappi) [reply index.html:80]: Okey THanks

File: index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+</head>
+<body>
+    <div class="header-section1-container">
+        <!-- header -->
+        <header class="header">
+            <div class="humburger">
+                <span></span>
+                <span></span>
+                <span></span>
+            </div>
```
Comment (tresorsawasawa) [inline index.html]: - [ ] As mentioned by the previous reviewer and as required for the project, kindly ensure that the `Hamburger` button is also visible at the `Home Page` of the `MOBILE` and it should work as expected. Right now, it is not displaying.
Screenshot from 2023-08-02 17-52-24

File: index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                    <p class="p">
+                      Get opportunities to network with TMS affiliates around the world,
+                      also after the summit.
+                    </p>
+                  </div>
+                </div>
+        
+                <div class="red-box">Join  TechMingle Summit 2023</div>
+                <div class="whole-program">SEE THE WHOLE PROGRAM</div>
+              </section>
```
Comment (tresorsawasawa) [inline index.html:134]: - [ ] Good job trying to fix the change mentioned by the previous reviewer. However, while you were fixing it, I noticed that you mistakenly damage the original design on the `DESKTOP`. I suggest you all of them and re-check before submitting to make sure all are correct.
### Template Design
Screenshot from 2023-08-02 18-03-43

### Your Design
Screenshot from 2023-08-02 17-51-17

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @gilberthappi 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (gilberthappi) [reply index.html:134]: Okey Thanks

File: index.html (hunk 3 above)
Comment (gilberthappi) [reply index.html]: Ookey Thanks

File: index.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
```
Comment (roshan-bajgain) [inline index.html:134]: - [ ] [Optional] This is small changes so I am giving it optional, Good job on making the changes please make sure that give the container certain height so It looks great.
### Your design
his
### After I give certain height to container
sa

**Review by roshan-bajgain — APPROVED** [review]
Hi @gilberthappi, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: index.html (hunk 5 above)
Comment (gilberthappi) [reply index.html:134]: Okey Thanks

**Pull request merged**